[TASK] Update property information (#1115)
TYPO3-Documentation/TYPO3CMS-Reference-Typoscript: Documentation/ContentObjects/Fluidtemplate/Index.rst

    Sets the extension name of the controller.

    ..  warning::

>         Up to TYPO3 11, this property could be used as an alternative to the `extensionName`

Reviewer: ```suggestion
        Up to TYPO3 v11, this property could be used as an alternative to the `extensionName`
```

        argument in the ViewHelpers :ref:`f:translate <t3viewhelper:typo3-fluid-translate>`
        and :ref:`f:uri.resource <t3viewhelper:typo3-fluid-uri-resource>`, provided that
        translations and public images were stored in the usual paths in your extension.

>         As of TYPO3 v12 this is no longer supported and it is recommended to use absolute keys

Reviewer: ```suggestion
        As of TYPO3 v12 this is no longer supported and it is recommended to use absolute keys,
```

>         or the `extensionName` argument plus key arguments in the ViewHelper.

Reviewer: ```suggestion
        or the `extensionName` argument plus key argument (for example, :html:`key="LLL:EXT:my_extension/Resources/Private/Language/locallang.xlf:myKey"`) in the ViewHelper.
```

Reviewer 2: The example should go after "use absolute keys".
